proto/Makefile

```diff
@@ -38,6 +38,12 @@ remove-go:
 go-descriptor:
 	DIR=$(GIT_TOP_DIR) DESCRIPTOR_PATH=$(GIT_TOP_DIR)/proto bash $(GIT_TOP_DIR)/tools/gen/descriptor.sh
 
+.PHONY: go-descriptor-check
+go-descriptor-check:
```

Comment:
Adding the dependency rule for check rules seems better, like the other places. What do you think?

E.g.
```sh
.PHONY: go-descriptor-check
go-descriptor-check: go-descriptor
```